src/extensionVariables.ts

```diff
@@ -62,6 +62,7 @@ export namespace ext {
     export let contextsTree: AzExtTreeDataProvider;
     export let contextsTreeView: TreeView<AzExtTreeItem>;
     export let contextsRoot: ContextsTreeItem;
+    export let isContextSwitchInProgress: boolean = false;
```

Comment:
Is it possible to make this a public-modifiable static variable on `LocalRootTreeItemBase` instead of an extension variable? It only has relevance to `LocalRootTreeItemBase`.